scripts/spack/packages/axom/package.py

```diff
@@ -528,35 +674,44 @@ def initconfig_package_entries(self):
         entries = []
         path_replacements = {}
 
+        # Validate the 'components' spec, which can selectively enable components
+        self._validate_component_deps()
```

Comment:
Should the validation happen in initconfig? i.e. after concretization and after all of axom's dependencies have been built.

Please let me know if there's an earlier place that this can be done.